### ISSUE TITLE
[Prefer Choco] only show available apps

### ISSUE THREAD

**Issue by MyDrift-user** [ChrisTitusTech/winutil#2775]
**Is your feature request related to a problem? Please describe.**

- some apps do not have a choco variant, these will not work.

**Describe the solution you'd like**

- hide apps. that do not have a choco variant if "prefer choco" is checked

**Comment by Marterich** [comment]
Another thought regarding the install logic would be, if we actually want to keep the automatic fallback from choco to winutil and vice versa now that we have a button to choose which one to use. I think the auto fallback could lead to situations like "i selected [choco|winutil] but the other one got downloaded and installed anyways without my consent"

**Comment by MyDrift-user** [comment]
I did think about it, however I think a simple popup "do you want to use winget/choco" right before falling back would be the trick here

**Comment by Marterich** [comment]
What do you think about a pop-up [retry installation with...] with a timer (10sec or so)  that defaults to no so that people who use a config for installing do not install a package manager they don't like

**Comment by MyDrift-user** [comment]
Good idea, something else that crossed my mind would include this but instead of a checkbox a dropdown auto/winget/choco

**Comment by tikacrabill** [comment]
What seems to be the problem? I know i had to do a factory reset on my
system and delete everything on my one desktop. Im new at this so and with
all your emails freaked me out

Tikacrabill Consulting Inc.
